client/src/components/BrandImage.js

```diff
@@ -0,0 +1,8 @@
+import styled from '@emotion/styled';
+
+const BrandImage = styled.img`
+  width: ${(props) => (props.LoginPage ? '100px' : '50px')};
+  margin-right: ${(props) => (props.LoginPage ? '15px' : '10px')};
```

Comment:
The BrandImage shouldn't know if 8t is used in the LoginPage.. Properties should be in camelCase

Reply:
you mean, the `props.LoginPage` should be renamed in something unspecific, like `props.big`?